test/integration/gridfs/gridfs.test.ts

```diff
@@ -115,5 +121,30 @@ describe('GridFS', () => {
       const createIndexes = commandStartedEvents.filter(e => e.commandName === 'createIndexes');
       expect(createIndexes).to.have.lengthOf(0);
     });
+
+    context('find(oid)', () => {
+      let findsStarted;
+
+      beforeEach(async function () {
```

Comment:
```suggestion
      beforeEach(function () {
```

Doesn't need to be async